nibabel/gifti/parse_gifti_fast.py

```diff
@@ -71,14 +70,12 @@ def read_data_block(encoding, endian, ordering, datatype, shape, data):
     return newarr
 
 
-class Outputter(object):
+class GiftiImageParser(XmlImageParser):
```

Comment:
I used the structure of this `Outputter` class, which is duplicated in CIFTI (#249), as the structure for an `XmlImageParser` class.